apps/www/src/examples/cards/components/GitHubCard.vue

```diff
@@ -26,7 +26,7 @@ import { Separator } from '@/lib/registry/new-york/ui/separator'
 
 <template>
   <Card>
-    <CardHeader class="grid grid-cols-[1fr_110px] items-start gap-4 space-y-0">
+    <CardHeader class="grid grid-cols-[minmax(0,1fr)_110px] items-start gap-4 space-y-0">
```

Comment:
If TailwindCSS add this 👇  style to their `reset` file

```css
* {
min-width: 0;
}
```

We can remove `minmax`?

Reply:
Yes, it will work fine, made some reproduction [codepen](https://codepen.io/greenwich/pen/oNVYpJm), just remove css and see that grid layout would shift from `container` block 🙂 

Maybe we will add this styles for default, or will wait TailwindCSS v3.5?

Reply:
Perfect.

Rebuild registry again then we can merge this 🎉

Reply:
So, we add this 👇 to our default styles? If yes, where I can find it, because I don't know codebase perfectly 😁 
```css
* {
    min-width: 0;
}
```

Reply:
@zernonia Sry for taging you too much

https://www.joshwcomeau.com/css/interactive-guide-to-flexbox/#the-minimum-size-gotcha-11

I can't decide of this change can you check if it's worth to add or we should wait until TailwindCSS add this change to their reset.css

Reply:
We will wait for Zernonia approve and then merge it 🫂

Reply:
Thanks for the awesome job guys! I'm looking at it now 😁

Reply:
Let's be safe for now and keep `minmax`. As we are not sure if and when TW will include min-width in their reset. 😁